typescript/web/src/connectors/repository/utils/list.ts

```diff
@@ -31,6 +31,12 @@ export const list =
   ): Promise<Entity[]> => {
     const table = await getTable();
     if (where) {
+      if ("id" in where && "in" in where.id) {
+        return (await table.bulkGet(where.id.in)).filter(
+          (entity) => entity !== undefined
+        ) as Entity[];
+      }
+
```

Comment:
Which specific case does this covers?

Reply:
It's used here: https://github.com/labelflow/labelflow/blob/5be069f91fcdabd84671f60ec3842daa55b883da/typescript/common-resolvers/src/image/index.ts#L150

Reply:
That's not exactly what I meant by which specific case 😅 
If the `in` keyword is a feature that Dexie is missing over Prisma, can we fix the problem more globally and support it closer to Dexie?